Automattic/wp-calypso
# WooCommerce: Add variation editing UI.

This adds the variations list table and UI for managing variation data like price, dimensions, weight, and stock. Closes #12770.

To test:
* Visit `http://calypso.localhost:3000/store/products/:site/add`
* Start typing in some variation types.
* Type in and toggle variation fields and make sure their values update.

<img width="1061" alt="screen shot 2017-05-09 at 8 48 52 am" src="https://cloud.githubusercontent.com/assets/689165/25859799/768dd9b6-3494-11e7-8e0d-769eb1156e4f.png">
